.appveyor.yml

```diff
@@ -57,9 +50,9 @@ before_test:
 - ps:  '[IO.File]::WriteAllLines([IO.Path]::Combine(${env:APPVEYOR_BUILD_FOLDER}, "test", "config.ini"), ${conf_ini}, ${utf8})'
 - ps:  move "build_msvc\${env:PLATFORM}\${env:CONFIGURATION}\*.exe" src
 test_script:
-- cmd: src\test_bitcoin.exe
-- ps:  src\bench_bitcoin.exe -evals=1 -scaling=0
+- cmd: src\test_bitcoin.exe -k stdout -e stdout 2> NUL
+- cmd: src\bench_bitcoin.exe -evals=1 -scaling=0 > NUL
```

Comment:
This will hide precious debug information in case of failure?

See for example https://github.com/bitcoin/bitcoin/pull/14955/commits/1a7cebc1cdd3c8fb8751fd21c782679081397ae2